src/cmds/scripts/pbs_habitat.in

```diff
@@ -238,14 +238,16 @@ upgrade_pbs_database() {
 		echo "Database version file: ${data_dir}/PG_VERSION not found, cannot continue"
 		return 1
 	fi
-
+	
+	sys_pgsql_ver=$(echo `${PGSQL_DIR}/bin/psql -V` | awk '{print $NF}' | cut -d '.' -f 1,2)
```

Comment:
We're checking whether psql is in the PATH in pbs_pgsql_env.sh, so it would error out there itself if it did not exist.